server.js

```diff
@@ -1,19 +1,56 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import topMusicData from "./data/top-music.json";
+const listEndpoints = require('express-list-endpoints')
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+//Should I use docker for below issue or what do I need to do to solve it?
+//I couldn't make the MONGO_URL work with the localhost-version - when I did that it stopped working locally
+const mongoUrl = process.env.MONGO_URL || "mongodb+srv://ylva87:hHwxpsqpR3V62DS9@cluster0.hsexulf.mongodb.net/?retryWrites=true&w=majority";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+
+const Song = mongoose.model('Song', {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+})
+
+const seedDatabase = async () => {
+  const data = topMusicData
+  Song.collection.insertMany(data, function(err, r) {
+  })
+}
+// seedDatabase()
+
+//switch to cover error-types and generate response
+const handleErrors = (err, res) => {
```

Comment:
This is the first time I've seen error handling in a function for itself. Maybe it would be more simple to just write them in the routes?

Reply:
Probably! I got some help from a more experienced developer but we didn't manage to make it work anyhow. So I changed it to be in the routes instaead and it worked much better.